projects/vir/src/app/+usage/component/data-table/data-table.component.ts

```diff
@@ -38,7 +39,7 @@ type TableType = 'downloads'|'people'|'user'|'userKeys'|'apiKeys';
           </laji-datatable>
       </div>
       <ng-template let-value="value" let-row="row" let-sort="sortFn" #downloadFileTpl>
-        <a [href]="'/api/file-download?id=' + value">{{ ('download.' + row.downloadType) | translate }}</a>
+        <a [href]="'/api/warehouse/download/secured/' + value + '?personToken=' + getPersonToken()">{{ ('download.' + row.downloadType) | translate }}</a>
```

Comment:
Using function calls in template code should be avoided, how about just bind the person token to an instance variable?